### PR TITLE
Add publication_list plugin to easily manage publication list.

### PR DESCRIPTION
**Pull request by xuhdev**
Add a plugin to manage publication list.

### CODE REVIEW

**Comment by xuhdev** [comment]
The reason the test failed on Travis CI is not caused by this PR. It is caused by something wrong with automated installation on dependency packages.

File: v7/publication_list/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Publication List
+
+A Nikola plugin to easily manage publication list.
+
+This plugin provides a directive `publication-list`, which reads a bib file and
+display the references in them on the web page. The `publication-list` directive
+accepts one option `:style:`, which is the style of the bibliography. All
+available styles are provided by Pybtex. You can see the [list of styles]() in
```
Comment (Kwpolska) [inline v7/publication_list/README.md]: Broken link.

    You can see the [list of styles][] in the Pybtex repository.

**Comment by Kwpolska** [comment]
The documentation (plugin description, README.md) needs a heavy sprinkling of the name **BibTeX**.

To make the test pass, merge/rebase on master, and fix two PEP 8 complaints:

```
v7/publication_list/publication_list.py:64:65: E251 unexpected spaces around keyword / parameter equals
v7/publication_list/publication_list.py:64:67: E251 unexpected spaces around keyword / parameter equals

**Comment by xuhdev** [comment]
@Kwpolska Done.

**Comment by Kwpolska** [comment]
Thanks for your contribution!

**Pull request merged**